### ISSUE TITLE
fit_position modifier for cover mode

### ISSUE THREAD

**Issue by impactfactory** [abwebdevelopers/oc-imageresize-plugin#38]
Fantastic Plugin would'nt know how to handle images anymore. Its great in combination with <picture>! Thank you so much for this great work.

Is the fit_position modifier for cover mode anyhow working? I don't how it should be done and nothing worked out so far..

Like this?

`{{ bg | media | resize(992, 300, { mode: 'cover', fit_position: '0 0' }) }}`

?

Or do I have to create a filter and combine mode: cover with an addition fit_positioning?

**Comment by bradietilley** [comment]
Yes ,it is meant for `cover` mode, and the available options any of the following: `top-left,top,top-right,left,center,right,bottom-left,bottom,bottom-right`. Defaults to `center`.

If you're still having issues let me know.

**Comment by impactfactory** [comment]
Thank you for the fast reply!

If we can make this work, would be cool to add your line to the documentation. 

With this code, resizing and format change work perfectly, but no effect regarding if the right part of the image stays visible...
```
<picture class="pagetitle-img">
        <source media="(max-width: 320px)" srcset="{{ bg | media | resize(320, 400, { mode: 'cover', fit_position: 'right', format: 'webp' }) }}">
etc.
</picture>
```
And I want to add, if you can make this work, your plugin is with the picture tag together the complete octobercms solution for responsive images....!

**Comment by impactfactory** [comment]
I guess this feature is not working. please test, correct me or cancel the feature out of the documentation :)

Im using this code

`<source type="image/webp" media="(max-width: 280px)" srcset="{{ post.img | media | resize(280, 547, { mode: 'cover', format: 'webp', fit_position: '{{ post.crop_area }}' }) }}">`

post.area is filled with top-right, top-left, bottom, etc. by the database via twig.

With not using the modifyer it crops on center. With using it crops on left and nothing else.
